databuilder/contracts/contracts.py

```diff
@@ -138,6 +146,8 @@ class WIP_Hospitalizations(TableContract):
 
 
 class WIP_HospitalizationsWithoutSystem(TableContract):
+    _name = "hospitalizations_"
```

Comment:
`hospitalizations_without_system` is a bit of a mouthful, and we can't call it `hospitalizations`, because there's another contract with the same name.

I think I'd thought that since `WIP_HospitalizationsWithoutSystem` isn't fixed, it wasn't worth coming up with a better name now.

Any suggestions?

Reply:
`hospitalizations_systemless`? I'd rather have a slightly unsatisfactory name that we hope to improve on (because this is WIP) than introduce what looks like a naming convention.

Reply:
I've gone with `hospitalizations_without_system` for consistency.